modules/code-editor/src/views/full/utils/index.ts

```diff
@@ -28,3 +28,9 @@ export const toastFailure = message =>
     intent: Intent.DANGER,
     timeout: 3500
   })
+
+export const sanitizeName = (text: string) =>
```

Comment:
Can we eventually move this in shared ? as we use this pretty much everywhere

Reply:
Yeah, it would definitely make sense to move it there, then remove usage everywhere else. I think there's some other utils we could do that with